docs/DEVELOPER.md

```diff
@@ -0,0 +1,61 @@
+Oengus developer environment
+============================
+
+This instruction provides steps on how to set up a development environment to work on Oengus backend in Ubuntu or other
+Debian-based GNU/Linux distributions. See [DOCKER.md](./DOCKER.md) for Docker instructions.
+
+Prerequisites
+-------------
+
+Install packages which are needed for running Oengus. JDK 16 or later is required. Just install the latest JDK available
+in your distribution. For example, for Java 17:
+
+    $ sudo apt install openjdk-17-jdk postgresql
+
+Source code
+-----------
+
+    $ git clone git@github.com:esamarathon/oengusio.git
+    $ cd oengusio
+
+Postgres
+--------
+
+If you haven't used Postgres before, set up a password for the connection to the database:
+
+    $ sudo su - postgres
+    postgres@hostname:~$ psql
+    postgres=# ALTER USER postgres PASSWORD '<your password here>';
+
+Create database `oengus`:
+
+    $ sudo su - postgres
+    postgres@hostname:~$ createdb oengus
+
+Discord integration
+-------------------
+
+Discord integration is needed to allow you to login into your Oengus dev environment of with your real Discord account.
```

Comment:
I would also mention the other services that are available

```suggestion
Discord integration is needed to allow you to login into your Oengus dev environment with your real Discord account.
```

Reply:
Done in 6e4bed45aaef42d75838ddb333e41a3c2d92fc2e.